Example/Tests/HttpFileParsingTests.m

```diff
@@ -49,6 +73,8 @@ - (void)verifyRequestWithURLString:(NSString *)urlString
 - (void)testNonexistentFileGives404
 {
     [self verifyRequestWithURLString:@"http://example.com/DoesntExist.html"
+                   additionalHeaders:nil
+                            bodyData:nil
```

Comment:
Would it make sense to have a convenience wrapper method like
```objc
- (void)verifyRequestWithURLString:(NSString *)urlString
                        completion:(void (^)(NSData *data, NSHTTPURLResponse *response, NSError *error))completion
{
     [self verifyRequestWithURLString:urlString
                    additionalHeaders:nil
                             bodyData:nil
                            completion:completion];
}
```
to avoid adding these two lines in a bunch of places?